### DISCUSSION TITLE
APA-Style bibliography: Need "et al." in italics.

### DISCUSSION THREAD

**Discussion by SquirlZz** [joaomlourenco/novathesis#440] [Q&A] [answered]
Greetings!

I'm doing my master's dissertation and I need help regarding the format of my citations/bibliography.

The citation style that I'm using is APA, and everything is working fine. However, whenever a citation occurs, I need the text "et al." to show in _italics_ . (i.e.: " Newton _et al._, 2023 " )
I've searched for a solution both in GitHub and Facebook novathesis support groups but was unsucessful in finding an answer. I've also found some external solutions to the problem, but i'm afraid I mess everything up, hence this support request.

How can i make this work?

Thank you in advance!

Technical details:
I'm using Overleaf as my LaTex editor ( [template V6.10.17] (2022-12-25) [English] )

**Comment by joaomlourenco** [comment]
Why do you want the “et al.” in italic?  Is is some weird regulation?  Or just because you _believe_ it is the right way to do it?
Normally, it **should not** be in italic…

**Reply by SquirlZz** [reply]
Thank you for the quick reply!

Well... Actually, my advisor, advised me to do so. But, having your clarification in mind, I'll definately talk to him about it, and try and figure out his reasoning behind it. I'll let you know if there's a weird regulation in the environmental engineering field!

**Comment by joaomlourenco** [comment, marked as answer]
Still, if you _really_ want to do that, add the following to the file `Config/5_packages.tex`
```latex
\usepackage{xpatch}

\xpatchbibmacro{name:andothers}{%
  \bibstring{andothers}%
}{%
  \bibstring[\emph]{andothers}%
}{}{}
```
